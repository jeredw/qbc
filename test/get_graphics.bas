get (0,0)-(16, 16), sprite%
get step(0,0)-step(16, 16), sprite%(2)
