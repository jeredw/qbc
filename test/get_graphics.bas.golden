(program 
  (statement 
    (get_graphics_statement get 
      ( 
        (expr 
          (literal 0)) , 
        (expr 
          (literal 0)) ) - 
      ( 
        (expr 
          (literal 16)) , 
        (expr 
          (literal 16)) ) , 
      (variable_or_function_call sprite%))) \n 
  (statement 
    (get_graphics_statement get step 
      ( 
        (expr 
          (literal 0)) , 
        (expr 
          (literal 0)) ) - step 
      ( 
        (expr 
          (literal 16)) , 
        (expr 
          (literal 16)) ) , 
      (variable_or_function_call sprite% 
        (args_or_indices 
          ( 
            (expr 
              (literal 2)) ))))) \n)
